src/index.js

```diff
@@ -100,8 +101,40 @@ const middleware = (config = {}) => (req, res, next) => {
         error: 'Signature verification failed!'
       })
     }
+    const certificates = JSON.parse(req.headers['x-authrite-certificates']).map(async cert => {
```

Comment:
Using `.map` here is non-ideal, because in case of an error, the loop will continue to run. We `return` out of the `.map`, but the middleware function would continue to run after the `.map` is complete, even in case of n error.